src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -145,4 +137,16 @@ IOktaApi oktaApi
 		return sourceCache.Where( session => session.ExpiresAt > currTime ).ToList();
 	}
 
+	private static bool IsMfaFactorTypeSupported( string mfaFactorType ) {
+		return mfaFactorType is
+			"call"
+			or "email"
+			or "question"
+			or "sms"
+			or "token:hardware"
+			or "token:hotp"
+			or "token:software:totp"
+			or "token";
```

Comment:
We currently support two kinds of MFA factors:
1. user enters a code/answer directly
2. we send a message (aka "challenge") to the user and the user enters a code (akak "challenge response") from that message

These all fall into the two kinds.